ta/tpm_log_test/include/ta_tpm_log.h

```diff
@@ -0,0 +1,12 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
```

Comment:
@jforissier, is this ok:
```c
/* SPDX-License-Identifier: BSD-3-Clause
 *
 * Copyright (...)
```
or prefer the below?
```c
/* SPDX-License-Identifier: BSD-3-Clause */
/*
 * Copyright (...)
```

Reply:
@etienne-lms good catch. Second form is preferred, in fact I don't think the first one passes checkpatch.